trim_lcnaf.rb

```diff
@@ -0,0 +1,23 @@
+File.open("updated_trimmed_lcnaf.yml", "a") do |trimmed_file|
+  File.open("lcnaf.skos.ndjson").each_line do |line|
+    line_hash = eval(line)
```

Comment:
Isn't parsing JSON in ruby done via 

```
require 'json'
JSON.parse(line)
```
? eval executes a string as though it was Ruby code no?

Reply:
I have to check again, but I think there was a reason I was not using `JSON.parse`. I think it was blowing up on the strings I was passing, but it's worth checking again as long as we're revisiting this.